src/SchemaVersion.cpp

```diff
@@ -41,6 +41,7 @@ bool SchemaVersion::apply(const QSqlDatabase &db)
         upgradeVer.prepare("update qtdbmigration_meta set cur_ver = cur_ver + 1");
         return upgradeVer.exec();
     } else {
+        qDebug() << "SqLite error:" << q.lastError().text() << ", SqLite error code:" << q.lastError().number();
```

Comment:
I would go with qWarning or even qError here.

Also note, we may use any other database engine here. So I would not use "Sqlite" term here.

Reply:
Set to qError. Term "Sqlite" replaced with "Database engine".